hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeAuthenticator.java

```diff
@@ -83,8 +80,9 @@ public default void setup(final Map<String, Object> config) {
     }
 
     @Override
-    public default User authenticate(final Map<String, String> credentials)
-                                     throws AuthenticationException {
+    default User authenticate(final Map<String, String> credentials)
+            throws AuthenticationException {
```

Comment:
can we keep the origin style: align 'throws' with 'final'

Reply:
#1996